examples/gcp/ingress/values_https.yaml

```diff
@@ -0,0 +1,17 @@
+# This example shows how to deploy and expose GraphDB with GCP Load Balancer Controller Ingress with SSL enabled.
+
+configuration:
+  # Change this to your Google Domains domain name or use nip.io which you can do after you deploy the chart since you
+  # need to map the public IP address of the GCP Ingress to the nip.io.
+  externalUrl: https://ip.address.nip.io/
+ingress:
+  enabled: true
+  className: gce
+  annotations:
+    # This line could be omitted as GKE controller processes any Ingress that does not have
+    # the annotation kubernetes.io/ingress.class, or has the annotation with the value gce
+    kubernetes.io/ingress.class: gce
+    # Defines the name of the static IP address which the Ingress should use
+    kubernetes.io/ingress.global-static-ip-name: ""
```

Comment:
Is this required? Cause it is missing from the other example.

Reply:
Using a static ip for the ingress resource is not required but it makes the configuration process much easier. It is missing in the other example since in this case when the ip is dynamic along with having to change the external URL you also have to update your managed certificate object to support the domain. The use of the static ip address in this example reduces the configuration overhead.

Reply:
Can you add a link to google's documentation about these annotations?

Reply:
https://cloud.google.com/kubernetes-engine/docs/how-to/managed-certs#prerequisites - Prerequisites for setting up TLS
https://cloud.google.com/kubernetes-engine/docs/concepts/ingress#controller_summary- Ingress controller behaviour
I have also mentioned these links in the README file